.codecov.yml

```diff
@@ -50,3 +48,6 @@ comment:
 
 ignore:
   - "libs"
+fixes:
+  - "testsuite/python/save_checkpoint*::testsuite/python/save_checkpoint.py"
+  - "testsuite/python/test_checkpoint*::testsuite/python/test_checkpoint.py"
```

Comment:
Could it be that, for Python coverage (in contrast to C++ coverage), this operation actually causes the last `save_checkpoint*` file (in alphabetic or timestamp order) to be used for the coverage report?  I'm trying to figure out why some parts of the checkpoint tests appear with no coverage.

Reply:
The `.coverage` file from the cuda9 artifact contains line coverage for the part that are red in codecov.

Reply:
yeah it seems like it overwrites previous reports

Reply:
offline discussion: let's discuss plans for a more maintainable checkpointing test framework on Monday meeting, that would solve this issue as a side-effect